src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -20,6 +20,7 @@
       <IncludeInNuget>true</IncludeInNuget>
     </ProjectReference>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
```

Comment:
>6.0.0" [](http://example.com/codeflow?start=82&length=6)

Is there a reason this dependency is hard coded and not using a value from our version file?

Reply:
Nope,, I will make a change to follow the pattern in Versions.props